src/tensor_ops/map.rs

```diff
@@ -35,6 +35,46 @@ pub fn relu<T: Tensor<Dtype = f32>>(t: T) -> T {
     map_df_uses_fx(t, |x| x.max(0.0), |fx| if fx > &0.0 { 1.0 } else { 0.0 })
 }
 
+/// [Leaky Rectified Linear Unit (Leaky ReLU)](https://en.wikipedia.org/wiki/Rectifier_(neural_networks)#Leaky_ReLU). `max(0.01 * t, t)`
+///
+/// The derivative is '1', for `t >= 0`, and `0.01` elsewhere.
+///
+/// Examples:
+/// ```rust
+/// # use dfdx::prelude::*;
+/// let t = tensor([-1.0, 0.0, 1.0, 2.0]);
+///
+/// // use function version
+/// let r = leakyrelu(t.clone());
+///
+/// // or the tensor method!
+/// let r2 = t.leakyrelu();
+/// ```
+pub fn leakyrelu<T: Tensor<Dtype = f32>>(t: T) -> T {
```

Comment:
This is a bit of a nit pick 😅  but lets keep this as two separate words with an underscore
```suggestion
pub fn leaky_relu<T: Tensor<Dtype = f32>>(t: T) -> T {
```

Reply:
Haha it’s an understandable nitpick, I went back and forth on the underscore. 

Thanks for the comments, I’ll fix them up!